src/porepy/models/geometry.py

```diff
@@ -77,7 +54,6 @@ def set_geometry(self) -> None:
 
     @property
     def domain(self) -> pp.Domain:
-        """Domain of the problem."""
         return self._domain
 
     def set_domain(self) -> None:
```

Comment:
This method, and set_fractures, set_well_network, are now clearly private to this mixin class. Should we make them so by an underscore prefix? I am not sure myself.

Reply:
The same question applies to several other methods, for instance in the data_saving_model_mixin.

Reply:
I'm not sure that's true for domain. I access it sometimes in my run scripts.

Reply:
True.